classes/tdx-encrypted.bbclass

```diff
@@ -0,0 +1,39 @@
+# Enable encryption support
+TDX_ENC_ENABLE = "1"
+
+# Encryption key backend
+# This variable defines how the encryption key is managed
+# Available options:
+#    cleartext -> key is stored in clear text (for testing purposes only!)
+TDX_ENC_KEY_BACKEND ?= ""
```

Comment:
Is the intent here to not put a default so that users must consciously make a decision on the configuration?

Reply:
Exactly. I don't think we should leave the cleartext backend enabled by default because it is insecure.

But as soon as we have the CAAM backend implemented, then I think we can set it by default in iMX based modules.

I also plan to write some documentation after implementing CAAM support.

Reply:
Makes sense to me.